docs/reference-1/credentials-1.md

```diff
@@ -0,0 +1,107 @@
+# Credentials
+
+AnyCloud deployments expect the cloud credentials to be configured using a local file that is not committed to any repository and is located at `~/.anycloud/credentials.json`. Each deployment will be keyed by a name and will provide the cloud provider configuration via two fields:  `credentials` and `cloudProvider`. Possible values for `cloudProvider` are `AWS`, `GCP` and `Azure`. `credentials` will have a different schema depending on which cloud provider it is.
+
+## AWS
+
+The schema for the AWS deployment config is as follows:
+
+```javascript
+{
+  "string": {
+    "cloudProvider": "string",
+    "credentials": {
+      "accessKeyId": "string",
+      "secretAccessKey": "string"
+    }
+  },
+  ...
+}
+```
+
+Follow the steps in this part of the Tutorial to get the `accessKeyId` and `secretAccessKey`. An example would look like:
+
+```javascript
+{
+  "stripe-aws": {
+    "cloudProvider": "AWS",
+    "region": "us-west-1",
+    "vmType": "t2.medium",
+    "credentials": {
+      "accessKeyId": "#####################",
+      "secretAccessKey": "###################"
+    },
+  },
+  ...
+}
+```
+
+## GCP
+
+The schema for a GCP deployment config is as follows:
+
+```javascript
+{
+  "string": {
+    "cloudProvider": "string",
+    "credentials": {
+      "privateKey": "string",
+      "clientEmail": "string",
+      "projectId": "string",
+    }
+  }
+  ...
+}
+```
+
+Take a look at the exported JSON file from your GCP [credentials]() and grab your `project_id`, `private_key` and `client_email`. An example of a GCP cloud configuration will look something like this:
+
+```javascript
+{
+  "stripe-gcp": {
```

Comment:
Why the name `stripe-*` on these? It's a bit confusing and makes me think you might need a Stripe account, too?

Reply:
Oh, that was just the name of an example company. I will use `piedpiper-*` 😂